src/Microsoft.OData.Edm/Validation/ValidationRules.cs

```diff
@@ -2497,6 +2497,22 @@ public static class ValidationRules
                     }
                 });
 
+        /// <summary>
+        /// Validates that if a vocabulary annotation doesn't have a value, it should declare a type with a default value.
+        /// </summary>
+        public static readonly ValidationRule<IEdmVocabularyAnnotationWithDefault> VocabularyAnnotationUseDefaultTermMustHaveDefaultValue =
+            new ValidationRule<IEdmVocabularyAnnotationWithDefault>(
+                (context, annotation) =>
+                {
+                    if (annotation.UsesDefault && annotation.Term.DefaultValue == null)

```

Comment:
>UsesDefault [](http://example.com/codeflow?start=35&length=11)

The logic here doesn't make sense to me. 

if (annotation.UsesDefault == true), it means annotation.Term has the default value. Otherwise, you can not create annotation. (because it will throw in 
EdmVocabularyAnnotation constructor).

So, it seems it's a conflict. Since we have the verification in the EdmVocabularyAnnotation constructor, we can consider remove this rule?  @mikepizzo

Reply:
Yes;  for *our* implementation of IEdmVocabularyAnnotationWithDefault, we will fail before creating the annotation if UsesDefault==true and the term has no default value.  I was okay with adding the rule because implementation of IEdmVocabularyAnnotationWithDefault may not have that logic, and I thought it would be a good exercise for Lisi to author a model validation rule.  However, I agree that, now that we have validation logic in EdmVocabularyAnnotation, we can remove the rule and save performance of running an extra rule.

Reply:
Removed ✅

Reply:
Wait a second; when we read XML CSDL, we'll create a CsdlSemanticsVocabularyAnnotation, and I don't think that currently validates that either a value is specified or that the term has a default value -- nor should it validate, since we don't want to throw exceptions for invalid annotations when reading CSDL.